pydropsonde/helper/__init__.py

```diff
@@ -303,9 +312,18 @@ def calc_iwv(ds, sonde_dim="sonde_id", alt_dim="alt"):
     iwv = typhon.physics.integrate_water_vapor(
         vmr[mask], pressure[mask], T=temperature[mask], z=alt[mask]
     )
-    ds_iwv = xr.DataArray([iwv], dims=[sonde_dim], coords={})
-    ds_iwv.name = "iwv"
-    ds_iwv.attrs = {"standard name": "integrated water vapor", "units": "kg/m^2"}
+
+    ds_iwv = xr.DataArray(
+        [iwv],
+        dims=[sonde_dim],
+        coords={},
+        name="iwv",
+        attrs=dict(
+            standard_name="atmosphere_mass_content_of_water_vapor",
```

Comment:
**TL;DR: I'd use this standard name as suggested and add a description similar to "vertically integrated water vapour between surface and aircraft altitude".**

This standard name seems to fit to the variable as far as I understand. The only thing that came to my mind on top of that is that we don't integrate over the *full* atmorspheric column as HALO is not flying at 80km altitude. I'd still use this standard name and add a description explaining that this quantity is calculated from the vertical integral from surface up to the aircraft height. 
Alternatively, we could use the more complicated standard name "mass_content_of_water_vapor_in_atmosphere_layer" and add the layer boundaries being surface and aircraft altitude. However, I think the latter is meant to describe data that have integrals over more than one layer which is what we have or want to have in the end.

Reply:
that's true... i would expect that above 14km not much happens regarding total water vapor content, as it is relatively cold and dry. but yes, theoretically it is not the whole water vapor content